src/App.jsx

```diff
@@ -1,13 +1,42 @@
-
+import { Route,Routes } from "react-router-dom"
+import Home from "./components/Home"
+import UserContext from "./contexts/UserContext"
+import { useEffect, useState } from "react"
+import ArticlesList from "./components/ArticlesList"
+import { getArticles , getTopics} from "./utils/api"
+import Header from "./components/Header"
 
 
 function App() {
-  
+  const [query,setQuery] =useState('')
+  const [articles,setArticles] = useState([])
+  const [topics,setTopics] =useState([])
+  const [loggedInUser,setLoggedInUser] = useState({
+    "username": "happyamy2016",
+    "name": "Amy Happy",
+    "avatar_url": "https://vignette1.wikia.nocookie.net/mrmen/images/7/7f/Mr_Happy.jpg/revision/latest?cb=20140102171729"
+  })
 
+  useEffect(()=>{
+    getArticles(query).then((body)=>{
+      setArticles(body)
+    })
+    getTopics().then((body)=>{
+      setTopics(body)
+  }).catch((err)=>{
+      console.log(err,'err');
+  })
```

Comment:
The catch here is only chained to the getTopics handler, you might need to chain these promises together so the catch will work for getArticles and getTopics